src/tasks/component-installers/cert-manager.ts

```diff
@@ -0,0 +1,184 @@
+                await this.kubeHelper.deleteServiceAccount(CA_CERT_GENERATION_SERVICE_ACCOUNT_NAME, CERT_MANAGER_NAMESPACE_NAME)
+                await this.kubeHelper.deleteRole('ca-cert-generator-role', CERT_MANAGER_NAMESPACE_NAME)
+                await this.kubeHelper.deleteRoleBinding('ca-cert-generator-role-binding', CERT_MANAGER_NAMESPACE_NAME)
+
+                await this.kubeHelper.deleteJob(CA_CERT_GENERATION_JOB_NAME, CERT_MANAGER_NAMESPACE_NAME)
+              } catch {
+                // Do nothing
+              }
+            }
+
+            // Wait until the secret is available
+            await this.kubeHelper.waitSecret('ca', CERT_MANAGER_NAMESPACE_NAME)
+          } else {
+            task.title = `${task.title}...already exists`
+          }
+        }
+      },
+      {
+        title: 'Set up Che certificates issuer',
+        task: async (_ctx: any, task: any) => {
+          const cheClusterIssuerExists = await this.kubeHelper.clusterIssuerExists('che-cluster-issuer')
+          if (!cheClusterIssuerExists) {
+            const cheCertificateClusterIssuerTemplatePath = path.join(flags.resources, '/cert-manager/che-cluster-issuer.yml')
+            await this.kubeHelper.createCheClusterIssuer(cheCertificateClusterIssuerTemplatePath)
+
+            task.title = `${task.title}...done`
+          } else {
+            task.title = `${task.title}...already exists`
+          }
+        }
+      },
+      {
+        title: 'Request self-signed certificate',
+        task: async (ctx: any, task: any) => {
+          if (ctx.cheCertificateExists) {
+            throw new Error('Che certificate already exists.')
+          }
+
+          const certificateTemplatePath = path.join(flags.resources, '/cert-manager/che-certificate.yml')
+          await this.kubeHelper.createCheClusterCertificate(certificateTemplatePath, flags.domain)
+          ctx.cheCertificateExists = true
+
+          task.title = `${task.title}...done`
+        }
+      },
+      {
+        title: 'Wait for self-signed certificate',
+        task: async (_ctx: any, task: any) => {
+          await this.kubeHelper.waitSecret(CHE_SECRET_NAME, flags.chenamespace, ['tls.key', 'tls.crt', 'ca.crt'])
+
+          task.title = `${task.title}...ready`
+        }
+      },
+      {
+        title: 'Add local Che CA certificate into browser',
+        task: async (_ctx: any, task: any) => {
+          const cheSecret = await this.kubeHelper.getSecret(CHE_SECRET_NAME, flags.chenamespace)
+          if (cheSecret && cheSecret.data) {
+            const cheCaCrt = Buffer.from(cheSecret.data['ca.crt'], 'base64').toString('ascii')
+            const cheCaPublicCertPath = path.join(os.homedir(), 'cheCA.crt')
+            fs.writeFileSync(cheCaPublicCertPath, cheCaCrt)
+
+            task.title = `❗[MANUAL ACTION REQUIRED] Please add local Che CA certificate into your browser: ${cheCaPublicCertPath}`
```

Comment:
Can we make MANUAL ACTION REQUIRED like colored to yellow?
![Screenshot_20200131_102126](https://user-images.githubusercontent.com/5887312/73523653-0577e680-4414-11ea-8a57-c1b3ad3a01bb.png)

Firefox allows me to import that file and it works like a charm!

But with the latest stable Chrome (Version 79.0.3945.130 (Official Build) (64-bit)) I got the following error
![Screenshot_20200131_102202](https://user-images.githubusercontent.com/5887312/73523714-23454b80-4414-11ea-9558-8802b6290024.png)

Reply:
Yes, will take a look if we can do something around coloring

Reply:
I have exactly the same version of Gooogle Chrome: 
![image](https://user-images.githubusercontent.com/15607393/73526102-255dd900-4419-11ea-96d1-d4b8baea8127.png)
And everything works like a charm:
![image](https://user-images.githubusercontent.com/15607393/73526219-67871a80-4419-11ea-8c11-42fc3bbf7f01.png)

Reply:
@sleshchenko can you please describe step-by-step how exactly your add the CA certificate into your Google Chrome browser?